coralproject/talk
# [CORL-2731] Add a message screen reader fix

## What does this PR do?
This PR fixes a bug in which a screen reader reads the word "Add" as "A D D" in the "Add message" buttons when configuring a stream.

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?
none.

## Does this PR introduce any new environment variables or feature flags?
No.

## If any indexes were added, were they added to `INDEXES.md`?
n/a

## How do I test this PR?
1. Enable a screen reader
2. As a moderator or admin, navigate to the "configure" tab of a stream, to the section where you can add a message at the top of the stream
3. Tab to the Add Message button, both with the editor open and closed (it will stay open after adding and then removing a message)
4. Observe that the screen reader correctly pronoucnes "ADD A MESSAGE"

## Where any tests migrated to React Testing Library?
No.

## How do we deploy this PR?
No special considerations needed.
